policyengine_us/parameters/gov/states/ky/tax/income/deductions/itemized.yaml

```diff
@@ -0,0 +1,16 @@
+description: Kentucky allows these itemized deductions.
+values:
+  2022-01-01:
+    - home_mortgage_interest
+    - investment_income_form_4952
+    - charitable_cash_donations
```

Comment:
should be `charitable_deduction` to capture the AGI limit (we can assume federal = KY AGI for this) but per the below, we don't need it if we list excluded deductions